### PR TITLE
feat: add Link atomic component

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-508
used ``ReactNode`` for ``text`` type since we might need to have an icon next to the link label

### CODE REVIEW

**Review by pietro-maximoff — APPROVED** [review]

File: packages/ui/src/1_atoms/Link/Link.types.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum LinkStyle {
+  primary = 'primary',
+  secondary = 'secondary',
```
Comment (creed-victor) [inline packages/ui/src/1_atoms/Link/Link.types.ts:3]: missing `tertiary` style

File: packages/ui/src/1_atoms/Link/Link.module.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  @apply text-primary-10;
+}
+
+.link.secondary {
+  @apply text-secondary-20;
+}
+
+.link.secondary:hover {
+  @apply text-secondary-30;
+}
```
Comment (creed-victor) [inline packages/ui/src/1_atoms/Link/Link.module.css]: I think css nesting is enabled for modules and can be used like this:

```suggestion

.link {
  @apply underline text-xs font-medium;
  
  &.primary {
    @apply text-primary-20;
    &:hover {
      @apply text-primary-10;
    }
  }
  
  &.secondary {
    @apply text-secondary-20;
    &:hover {
      @apply text-secondary-30;
    }
  }
}
```

File: packages/ui/src/1_atoms/Link/Link.module.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import '../../../styles/index.css';
```
Comment (creed-victor) [inline packages/ui/src/1_atoms/Link/Link.module.css]: Is there some kind of issues if file is not imported?

File: packages/ui/src/1_atoms/Link/Link.types.ts (hunk 1 above)
Comment (soulBit) [reply packages/ui/src/1_atoms/Link/Link.types.ts:3]: design only has primary/secondary, spec was out of date - I updated it now

**Review by soulBit — APPROVED** [review]
Looks good if other review comments are resolved

**Review by pietro-maximoff — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by LunaSov — APPROVED** [review]

**Pull request merged**